public/inngest-sdk.js

```diff
@@ -148,7 +148,29 @@
     return errors;
   }
 
-  function context() {
+  const utmKeys = [
+    "utm_source",
+    "utm_campaign",
+    "utm_content",
+    "utm_medium",
+    "utm_term",
+    "ref",
+  ];
+
+  function context(eventData) {
+    const utmProps = {};
+    const isFirstTouch = eventData.first_touch;
+    try {
+      const urlParams = new URLSearchParams(window.location.search);
+      // we track both first touch UTMs props and all touchpoints UTMs
+      for (let utmKey of utmKeys) {
+        utmProps[utmKey] = urlParams.get(utmKey);
+        if (isFirstTouch) {
+          utmProps[`first_${utmKey}`] = urlParams.get(utmKey);
+        }
+      }
+    } catch (e) {}
+
```

Comment:
This code looks duplicative of the code after defining `var data`  and this line. We should remove that other UTM parsing code as future changes to this code would likely be overwritten by the other code after this if statement:

```js
if (window && window.location.search.length) {
```

Reply:
Good catch! 🦅 
Solved in [72508fb](https://github.com/inngest/website/pull/991/commits/72508fbc7c6303ec883d0717dac9c402f1ee47c6)